server/src/infra/repositories/media.repository.ts

```diff
@@ -23,10 +25,25 @@ export class MediaRepository implements IMediaRepository {
   }
 
   async resize(input: string | Buffer, output: string, options: ResizeOptions): Promise<void> {
-    await sharp(input, { failOn: 'none' })
+    let colorspace = options.colorspace;
+    if (options.colorspace !== Colorspace.SRGB) {
+      try {
+        const { space } = await sharp(input).metadata();
+        // if the image is already in srgb, keep it that way
+        if (space && space === Colorspace.SRGB) {
+          colorspace = Colorspace.SRGB;
```

Comment:
This is confusing me. This whole block is essentially a no-op?

Reply:
Perhaps renaming the variables to `targetColorSpace` and `actualColorSpace` or similar would make it more clear. I believe this will ignore the targetColorSpace argument in the case the file is srgb.

Reply:
I changed the local variable to `colorProfile`, since this is what it actually sets in the sharp pipeline.